arquillian-ape-sql/standalone/dbunit/src/main/java/org/arquillian/ape/rdbms/dbunit/DbUnitOptions.java

```diff
@@ -0,0 +1,100 @@
+package org.arquillian.ape.rdbms.dbunit;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class DbUnitOptions implements Map<String, Object> {
```

Comment:
This is for standalone mode, and configuration is programmatically provided. In case of NoSQL databases we are doing in that way, so I would suggest to use the same approach (At least for the none annotation style)